api/controllers/CheckUploadedDocumentsController.js

```diff
@@ -17,6 +21,32 @@ const CheckUploadedDocumentsController = {
         });
     },
 
+    async _addUserDataToDB(req) {
```

Comment:
This block looks like it's duplicated in ApplicationTypeController.newApplication.  Consider separating it out to a function in a helper class that can then be referenced in both controllers.